### DISCUSSION TITLE
About L14 vgg

### DISCUSSION THREAD

**Discussion by zzzyzh** [rasbt/stat453-deep-learning-ss21#7] [General]
Hi!
I'd like to ask why should we use avgpool instead pf maxpool?

**Comment by rasbt** [comment]
That's a good question. I just see that the original VGG used maxpooling rather than average pooling in the output layer. I can't remember exactly, but I may have compared to the two and found that average pooling performed better here in the output layer. Adaptive, global, or generally average pooling in the output layer is quite common to not throw out too much information before the fully connected layer.
